pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = commune.settings
+django_debug_mode = true
+python_files = tests.py test_*.py
```

Comment:
add newline at the end of the file

Reply:
fixed it with second commit

Reply:
are you sure? it still looks like there's no newline

Reply:
check the last commit i made

Reply:
also it wont pass the flake8 if i didnt fix it..